code/src/components/ThoughtItem.css

```diff
@@ -0,0 +1,130 @@
+.thought-item-container {
+    display: flex;
+    flex-direction: column;
+    border: none;
+    box-sizing: content-box;
+    background-color: rgba(185, 198, 233, 0.781);
+    padding: 10px;
+    width: 45vh;
+    border-radius: 10px;
+}
+
+.thought-message {
+    word-break: break-word;
+    font-size: 17px;
+    color: rgb(42, 45, 54);
+    font-weight: bold;
+}
+
+.button-date-container {
+    display: flex;
+    justify-content: space-between;
+    color: rgba(42, 45, 54, 0.637);
+    font-weight: bold;
+}
+
+.hearts-button-container {
+    display: flex;
+    align-items: center;
+    column-gap: 5px;
+}
+
+.like-button {
+    border-radius: 50%;
+    border: none;
+    background-color: rgb(226, 224, 250);
+    cursor: pointer;
+    height: 35px;
+    width: 35px;
+    transition: .2s;
+}
+
+.like-button:hover {
+    box-shadow: 0 0 10px rgba(42, 45, 54, 0.637);
+}
+
+.like-button:active {
+    -moz-transform: scale(0.98);
+    -webkit-transform: scale(0.98);
+    transform: scale(0.98);
+    box-shadow: 0 0 1px rgb(42, 45, 54);
+}
+
+.heart-icon {
+    width: 85%;
+    margin-top: 5px;
+}
+
+.heart-icon:hover {
+    animation: pulse 1.5s infinite;
+    animation-timing-function: linear;
+}
```

Comment:
Really nice animation!

If you want you could make the heart animation when you hover the parent like this. (It will still only make the animation for the heart but you can se the animation if you hover the parent) :

```
like-button:hover heart-icon {
    animation: pulse 1.5s infinite;
    animation-timing-function: linear;
}
```